tests/ftest.sh

```diff
@@ -19,11 +19,19 @@ PLATFORM='unknown'
 MAX_RSS="200M"
 MAX_DURATION=600
 
+export PERF8_TRACE=${PERF8_TRACE:-False}
 export REFRESH_RATE="${REFRESH_RATE:-5}"
 export DATA_SIZE="${DATA_SIZE:-medium}"
 export RUNNING_FTEST=True
 export VERSION='8.10.0-SNAPSHOT'
 
+if [ "$PERF8_TRACE" = true ]; then
```

Comment:
TIL, `=` is also used for comparison. From this [answer](https://stackoverflow.com/a/20449556), `=` is prefered, WDYT if we change all `==` to `=`?

Reply:
Good call, I've changed it to `==` for consistency within the file, but we might wanna move to `=` everywhere else.